drivers/flash/spi_flash_at45.c

```diff
@@ -17,10 +19,19 @@ LOG_MODULE_REGISTER(spi_flash_at45, CONFIG_FLASH_LOG_LEVEL);
 /* AT45 commands used by this driver: */
 /* - Continuous Array Read (Low Power Mode) */
 #define CMD_READ		0x01
+#define CMD_READ_HF		0x0B
```

Comment:
Add comment explaining the ID, the same way the CMD_READ is explained.

Reply:
fixed